polaris/ocean/convergence/spherical/analysis.py

```diff
@@ -352,6 +348,39 @@ def exact_solution(self, mesh_name, field_name, time, zidx=None):
 
         return ds_init[field_name]
 
+    def get_output_field(self, mesh_name, field_name, time, zidx=None):
+        """
+        Get the model output field at the given time and z index
+
+        Parameters
+        ----------
+        mesh_name : str
+            The mesh name which is the prefix for the output file
+
+        field_name : str
+            The name of the variable of which we evaluate convergence
+
+        time : float
+            The time at which to evaluate the exact solution in seconds
+
+        zidx : int, optional
+            The z-index for the vertical level to take the field from
+
+        Returns
+        -------
+        field_mpas : xarray.DataArray
+            model output field
+        """
+        ds_out = xr.open_dataset(f'{mesh_name}_output.nc')
+
+        tidx = _time_index_from_xtime(ds_out.xtime.values, time)
+        ds_out = ds_out.isel(Time=tidx)
+
+        field_mpas = ds_out[field_name]
+        if zidx is not None:
+            field_mpas = field_mpas.isel(nVertLevels=zidx)
+        return field_mpas
```

Comment:
@cbegeman, I need to make this a method similar to `exact_solution()` because I need a way to calculate the water-column thickness (which is not an output variable).
1. Would you prefer a different name for the method?
2. Are you okay with this proposed approach?

Reply:
@xylar That looks good to me. My original thought was just that folks would add whatever variables they need to the output netcdf file but it's probably better this way so we don't modify those files.

Reply:
Thanks for the feedback!

Yeah, I don't think modifying files that a step doesn't "own" is a good idea and it's not possible to modify the files in the forward step itself because it's a "command-line" step that just calls the model (no python code allowed).